src/features/analysis/shared/parsing/data-parser.ts

```diff
@@ -136,13 +137,18 @@ function extractKeyStatsFromFields(fields: Record<string, GameRunField>): {
     tier = match ? parseInt(match[1], 10) : 0;
   }
   tier = tier || 0;
-  
+
+  const realTime = (fields.realTime?.value as number) || 0;
+  const gameTime = (fields.gameTime?.value as number) || 0;
+  const gameSpeed = realTime === 0 ? null : Math.round((gameTime / realTime) * 1000) / 1000;
```

Comment:
Extract a common function to calculate this and use it in this file and in csv-field-mapping.ts since it seems this is exactly the same calc, and there is quite a bit of conditional logic here.